app/views/candidate_interface/new_references/review/show.html.erb

```diff
@@ -0,0 +1,18 @@
+<% content_for :title, t('page_titles.new_references') %>
```

Comment:
We need to include the error prefix here, if the "Have you completed this section?" is in error.